packages/metrics-dotnet/MutationMetrics/Mutation.Metrics.Core.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Since this package should have zero dependencies I think we can target .NET Standard 2.0 without issue and achieve the highest level of compatibility.